kernel/update_kernel.sh

```diff
@@ -26,7 +26,7 @@ KR_REL=https://www.kernel.org/releases.json
 KR_SHA=https://cdn.kernel.org/pub/linux/kernel/v"${KR_SERIES}"/sha256sums.asc
 
 GENERATED_FILES=(linux-container.dsc linux-container.spec _service config)
-STATIC_FILES=(debian.dirs debian.rules debian.compat debian.control debian.copyright)
+STATIC_FILES=(debian.dirs debian.rules debian.compat debian.control debian.copyright debian.series)
```

Comment:
I don't understand why this is on the same commit? If it's not strictly related to the libelf change, could you add a separate commit for this change please?

Reply:
@jodh-intel Sure, I will send another PR for that, my bad. That's a missing change that I forgot to ask in the previous kernel PR, as it was merged.

Reply:
No probs, and thanks ;)